integration/log_integration_test.go

```diff
@@ -45,9 +46,23 @@ type testParameters struct {
 	sequencingPollWait  time.Duration
 }
 
+type consistencyProofParams struct {
+	firstTreeSize  int64
```

Comment:
Done.  I'm overall unimpressed with Go style and the seeming need to make it differ from every other language. They can get off my lawn.